misc/config_tools/xforms/misc_cfg.h.xsl

```diff
@@ -20,13 +20,22 @@
     <!-- Header include guard -->
     <xsl:value-of select="acrn:include-guard('MISC_CFG_H')" />
 
+    <xsl:apply-templates select="board-data/acrn-config" />
+
     <xsl:apply-templates select="config-data/acrn-config" />
```

Comment:
Shall we put the change to board_info.h.xsl? The maximum physical-address width should be board specific data, not a configurable data.

Reply:
Agree and update applied.